plugins/modules/database/misc/redis_data_info.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Andreas Botzner <andreas at botzner dot com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: redis_data_info
+short_description: Get value of key in Redis database
+description:
+  - Get value of keys in Redis database
```

Comment:
```suggestion
  - Get value of keys in Redis database.
```
Descriptions should always have complete sentences. (The exception is `short_description`.)